src/handlebars/releases.handlebars

```diff
@@ -39,22 +39,29 @@
       <h2 id="latest-select-text" style="margin-bottom: 0.5rem;">Select a platform</h2>
       <br>
 
-        <div id="latest-selector">
-          <script id="template-selector" type="text/x-handlebars-template">
-            {{! Create each platform's selector tile }}
+      <div id="latest-selector">
+        <script id="template-selector" type="text/x-handlebars-template">
+          {{! Create each platform's selector tile }}
             \{{#each releases}}
               <div class="latest-asset latest-grid-border" id='latest-selector-\{{platform_name}}' onclick='selectLatestPlatform("\{{platform_name}}")'>
                 <img alt='\{{platform_official_name}} logo' src='\{{platform_logo}}' />
                 <h2 class="officialName"><var platform-name>\{{platform_official_name}}</var></h2>
                 <p class="releaseName"><var release-name>\{{release_name}}</var></p>
               </div>
             \{{/each}}
+            <a id="docker_link" href="" target="_blank">
+            <div class="latest-asset latest-grid-border" id='latest-selector-Docker'>
```

Comment:
What happens if there's no valid docker link for this release?

Reply:
I'm pretty sure there's an open issue saying that we're missing some right now.

Reply:
this might help us spot those issues then